examples/HydroValleys/case3.jl

```diff
@@ -32,35 +32,35 @@ data[1]
 params = set_param( stages = 12, 
                     model_constructor_grid  = DCPPowerModel,
                     post_method             = PowerModels.post_opf,
-                    solver                  = ClpSolver());
+                    optimizer               = Clp.Optimizer);
 params
 
 #' ## Build Model
 m = hydrothermaloperation(data, params);
 
 #' ## Solve
-status = solve(m, iteration_limit = 60);
-status
+status = SDDP.train(m.policygraph;iteration_limit = 100);
 
 #' ## Simulation
-srand(1111)
-results = simulate_model(m, 100);
+import Random
+Random.seed!(1111)
+results = HydroPowerModels.simulate(m, 100);
 results
 
 #' ## Testing Results
 #' Objective
-using Base.Test
-@test isapprox(results["simulations"][1]["objective"], 12400.00, atol=1e-2)
+using Test
+@test isapprox(sum(s[:stage_objective] for s in results[:simulations][1]), 9400.0, atol=1)
```

Comment:
Why did the value (12400) change? Is it because you run more iterations? (I can't guarantee that SDDP.jl is bug-free, so changes like this are worth investigating.)

Reply:
It doesn't seem to be because I run more iterations. I augmented it because with 60 iterations, in the new framework, the sddp was not returning a stable solution.

My best guess is that the scenarios are not being sampled in the same way. I tried to seed it, but no luck.  Do you believe this could be the reason?

Reply:
Oh, it's probably because I turn cut selection on by default now. You can set the `cut_deletion_minimum` argument in [train](https://odow.github.io/SDDP.jl/latest/apireference/#SDDP.train) to a large number like `1_000_000` if you want to turn it off.